ext/native/thin3d/VulkanRenderManager.cpp

```diff
@@ -895,10 +950,20 @@ void VulkanRenderManager::BeginSubmitFrame(int frame) {
 void VulkanRenderManager::Submit(int frame, bool triggerFence) {
 	FrameData &frameData = frameData_[frame];
 	if (frameData.hasInitCommands) {
+		if (gpuProfilingEnabled_ && triggerFence) {
+			// Pre-allocated query ID 1.
+			vkCmdWriteTimestamp(frameData.initCmd, VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT, frameData.timestampQueryPool_, 1);
```

Comment:
Hm, ought we check the initCmd timing for subsequent submits in the frame (triggerFence = false)?

-[Unknown]

Reply:
Yeah probably, and we should subtract the time spent between the submits and show that separately. Thought I'd leave that for later.